bleach/html5lib_shim.py

```diff
@@ -248,24 +263,33 @@ def __iter__(self):
                     token['data'] = self.stream.get_tag()
                     token['type'] = CHARACTERS_TYPE
 
-                    # Yield the adjusted token
+                    last_error_token = None
                     yield token
 
+                elif token['type'] == PARSEERROR_TYPE:
+                    # If the token is a parse error, then let the last_error_token
+                    # go, and make token the new last_error_token
+                    yield last_error_token
+                    last_error_token = token
```

Comment:
I had to add this section because what's really going on is that the `HTMLTokenizer` raises two error tokens. So we needed to add handling for the case where `last_error_token` and `token` are both error tokens.